src/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import {
+  commitMutation,
+  createFragmentContainer,
+  graphql,
+  RelayProp,
+} from "react-relay"
+import { FollowButton } from "./Button"
+import * as Artsy from "../Artsy"
+
+export interface Props
+  extends RelayProps,
+    React.HTMLProps<FollowArtistButton>,
+    Artsy.ContextProps {
+  relay?: RelayProp
```

Comment:
I see you export this interface, but didn’t see where it was being used. I’d suggest not exporting an interface that has the `relay` prop, because that’s normally really a detail internal to the component.